geniza/templates/footer.html

```diff
@@ -5,9 +5,11 @@
 <nav id="footer-nav">
     <ul role="menu">
         <li class="menu-item" role="none">
-            {% pageurl site.root_page.localized as root_url %}
+            {% if skip_nav != "true" %}
```

Comment:
minor, but it looks a little strange to check for `"true"`. Could we do `skip_nav=1` instead when calling the template and then this could just check `if not skip_nav` ? Or will that not work if skip nav is undefined?

Reply:
You're right! It works fine like that, done in d74e959 👍